.changesets/feat_max_headers.md

```diff
@@ -0,0 +1,12 @@
+### Experimental Configuration Options for HTTP/1 Max Headers and Buffer Limits ([PR #6194](https://github.com/apollographql/router/pull/6194))
```

Comment:
My feedback on this block:

* This changelog can be renamed to be prefixed with `exp_` (instead of `feat_`; see [docs](https://github.com/apollographql/router-private/blob/17e88fac8f0184f90d8757183841920b85203673/.changesets/README.md?plain=1#L33)) and it will automatically show up in an "Experimental" section.  I think that's what it's intended for.
* This changelog is excellent in what it says, but I would _also_ update it to specify **WHY** the feature is marked as experimental (if my understanding is correct, it's not because it's unstable, but because we are waiting on the upstream specificity of whether option ultimately applies to HTTP v1 or HTTP/2, is that, correct?)
* I think in the docs we should say, "If you are building a custom binary, you will need to use a custom `cargo patch` entry."

Thoughts?

Reply:
All points are correct 👍